code/__DEFINES/dcs/signals.dm

```diff
@@ -930,6 +930,18 @@
 /// called on implants, after an implant has been removed: (mob/living/source, silent, special)
 #define COMSIG_IMPLANT_REMOVED "implant_removed"
 
+/// called as a mindshield is implanted: (mob/user)
+#define COMSIG_PRE_MINDSHIELD_IMPLANT "pre_mindshield_implant"
+	/// Did they successfully get mindshielded?
+	#define COMPONENT_MINDSHIELD_PASSED (1<<0)
+	/// Did they resist the mindshield?
+	#define COMPONENT_MINDSHIELD_RESISTED (1<<1)
```

Comment:
The first one should just be NONE, and the second should be (1<<0)
returning a bitflag to indicate that it was not blocked is a little redundant, especially if anyone tries to use that as a feature for some reason